frontend/device/pocketbook/device.lua

```diff
@@ -354,6 +354,24 @@ function PocketBook:initNetworkManager(NetworkMgr)
     function NetworkMgr:isWifiOn()
         return band(inkview.QueryNetwork(), C.NET_CONNECTED) ~= 0
     end
+
+    local UIManager = require("ui/uimanager");
+
+    local function wifiKeepAlive()
+        logger.dbg('check keep alive')
+
+        if (NetworkMgr:isWifiOn()) then
+            logger.dbg('ping wifi keep alive')
+
+            inkview.NetMgrPing();
+        end
+
+        -- Make sure only one wifiKeepAlive is scheduled
+        UIManager:unschedule(wifiKeepAlive)
```

Comment:
It looks like this will always run? Is that intentional? (It's probably not a big deal, but still. :-)

Reply:
Er, the function I mean, not this line itself. I commented here because I was kind of expecting to see a start and a stop at some other point as well.

Reply:
@Frenzie I will testing starting the keep-alive by scheduling it when wifi is enabled and not rescheduling it when the wifi connection is gone. This would give the best of both worlds of not using cpu cycles when wifi is off and making sure the process is self cleaning while avoiding duplicates.